sdk/eventgrid/azure-eventgrid/azure/eventgrid/_publisher_client.py

```diff
@@ -37,12 +53,36 @@ class EventGridPublisherClient(object):
 
     def __init__(self, topic_hostname, credential, **kwargs):
         # type: (str, Union[AzureKeyCredential, EventGridSharedAccessSignatureCredential], Any) -> None
-
         topic_hostname = _get_topic_hostname_only_fqdn(topic_hostname)
 
         self._topic_hostname = topic_hostname
+        self._client = EventGridPublisherClientImpl(
+            policies=EventGridPublisherClient._policies(credential, **kwargs),
+            **kwargs
+            )
+
+    @staticmethod
+    def _policies(credential, **kwargs):
+        # type: (Union[AzureKeyCredential, EventGridSharedAccessSignatureCredential], Any) -> List[Any]
         auth_policy = _get_authentication_policy(credential)
-        self._client = EventGridPublisherClientImpl(authentication_policy=auth_policy, **kwargs)
+        sdk_moniker = 'eventgridpublisherclient/{}'.format(VERSION)
```

Comment:
Should be "eventgrid", the previous one was incorrect as well:
https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy